src/core/cpu/arm.hpp

```diff
@@ -36,6 +36,7 @@ struct PSR_Flags
     bool carry;
     bool overflow;
     bool q_overflow;
+    bool ge[4];
```

Comment:
You should expose the GE bits in PSR_Flags::get/set so that they can be properly saved and restored when an exception occurs.